FluentPDO/FluentUtils.php

```diff
@@ -23,4 +23,23 @@ public static function formatQuery($query) {
 		$query = preg_replace("/\s+\n/", "\n", $query);
 		return $query;
 	}
-}
\ No newline at end of file
+
+    /** Fills in query placeholders with their parameter values; useful only for debugging.
+     * @param string $query
+     * @param array parameters
+     */
+    public static function populate($query, array $parameters) {
+        foreach ($parameters as $parameter_value) {
+            // Quoting mechanism.
+            if(!in_array($parameter_value, array(
```

Comment:
You're redefining the array on every iteration. Bad idea, move the array outside the loop.